src/Input/BoolParam.php

```diff
@@ -27,6 +29,11 @@ public function __construct(string $name)
     public function getQuestion(): Question
     {
         $default = $this->getDefault();
+        Assert::boolean($default, sprintf(
+            'Default value MUST be a boolean; received "%s"',
+            get_debug_type($default)
```

Comment:
I think this only works on PHP 8.0+, or do you have a polyfill installed?

If so, the polyfill must be explicitly declared in `composer.json`

Reply:
Yes, I have the polyfill installed.

Reply:
To clarify: it's in `composer.json`.